libs/payments/ui/src/lib/client/components/CheckoutForm.ftl

```diff
@@ -0,0 +1,11 @@
+## Checkout Form
+
+next-new-user-submit = Subscribe Now
+
+# Label for the Full Name input
+payment-name-label = Name as it appears on your card
```

Comment:
Should this have a `next-` prefix like the others?

Reply:
Not in this case. Strings prefixed with `next-` are existing `fxa-payments-server` strings that we'll be reusing in `payments-next`. In this case `payment-name-label` is a new string.